openpower/package/openpower-pnor/openpower-pnor.mk

```diff
@@ -61,6 +61,24 @@ SBE_BINARY_DIR = $(STAGING_DIR)/sbe_binaries/
 OPENPOWER_PNOR_SCRATCH_DIR = $(STAGING_DIR)/openpower_pnor_scratch/
 OPENPOWER_VERSION_DIR = $(STAGING_DIR)/openpower_version
 OPENPOWER_MRW_SCRATCH_DIR = $(STAGING_DIR)/openpower_mrw_scratch
+OUTPUT_BUILD_DIR = $(STAGING_DIR)/../../../build/
+OUTPUT_IMAGES_DIR = $(STAGING_DIR)/../../../images/
+
+FILES_TO_TAR = $(OUTPUT_BUILD_DIR)/hostboot-*/img/hbicore.bin.modinfo \
```

Comment:
Can we just grab everything in the hostboot-build-images directory instead of these individual files? That way if somebody in the future indicates a new necessary file for debug (see attrInfo.csv from a different story), it would just be picked up automatically here.